src/components/Login/Login.js

```diff
@@ -26,9 +26,14 @@ const fieldNames = {
   password: 'password',
 };
 
+function extractRedirect(location) {
```

Comment:
What do you think about 

```suggestion
function extractPageFromRedirect(location) {
```

You can also add a comment with the explanation that it is a patch, the reason, etc.

Reply:
fixed